view/frontend/web/js/view/payment/method-renderer/adyen-cc-method.js

```diff
@@ -132,19 +166,31 @@ define(
              * set up the installments
              */
             renderCCPaymentMethod: function() {
+                let componentConfig = this.buildComponentConfiguration();
+
+                this.cardComponent = adyenCheckout.mountPaymentMethodComponent(
+                    this.checkoutComponent,
+                    'card',
+                    componentConfig,
+                    '#cardContainer'
+                )
+
+                return true
+            },
+
+            buildComponentConfiguration: function () {
                 let self = this;
-                if (!self.getClientKey) {
+
+                if (!this.getClientKey) {
                     return false;
                 }
 
-                self.installments(0);
-
-                // installments
-                let allInstallments = self.getAllInstallments();
+                this.installments(0);
```

Comment:
@candemiralp I'm getting error from template `cc-form.html` template related to undefined variable, if installments aren't set to 0 in `defaults`

```
            defaults: {
                template: 'Adyen_Payment/payment/cc-form',
                installment: '', // keep it until the component implements installments
                installments: 0,
                orderId: 0, // TODO is this the best place to store it?
                storeCc: false,
                modalLabel: 'cc_actionModal'
            },
```

Reply:
Hello @Januszpl,

I will get back to you after checking this point.

Best Regards,
Can